src/libexpr/value.hh

```diff
@@ -38,6 +39,7 @@ class XMLWriter;
 
 
 typedef long NixInt;
+typedef double NixFloat;
```

Comment:
I can imagine cross-compiling 32-bit from a 64-bit machine will end up creating values that won't be computable on the 32-bit machine.